features/_your_results/period_percent_table.feature

```diff
@@ -0,0 +1,16 @@
+Feature:
+  As a user who has just completed the WPCC tool, I should be able to see helpful information on how the yearly increments is done.
```

Comment:
Please can we name the Feature. Normal convention would be to use the same naming as the file. But whatever gives a quick understanding of what the feature is testing. An example from elsewhere:
```
Feature: calculating eligible salary
```
Please break the feature description onto multiple lines. I have started trying to get all features to follow the format of:
```
In order to..
As a...
I want ...
```
This is completely open for discussion, but would be good to try to keep a similar format across all